src/components/Icons.tsx

```diff
@@ -0,0 +1,15 @@
+import HTMLComment from 'react-html-comment';
```

Comment:
TIL React doesn't support html comments but the license requires the attribution to be embedded.

Reply:
hmm I've always used the fontawesome react package - does that not include some alternative?

Reply:
I haven't used that, maybe it does something similar... I was worried about bundle size with that but I see they say importing the exact icons you need will let tree shaking do it's thing, so maybe that's a better alternative than this one-off package. I'll see what the build size is with the font awesome package when I get a chance.

Reply:
Or yeah from our convo if you want to pull font awesome in instead that would be great!